configs/nosite-x86_64/variables.yaml

```diff
@@ -4,9 +4,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 variables:
-  batch_time: ''
-  mpi_command: 'mpirun -n {n_nodes} -c {n_ranks} --oversubscribe'
-  batch_submit: '{execute_experiment}'
-  batch_nodes: ''
-  batch_ranks: ''
-  batch_timeout: ''
+  mpi_command: 'mpirun -n {n_ranks} --oversubscribe'
+  n_ranks: '7'
+  n_nodes: '7'
+  n_threads: '7'
```

Comment:
On CPU, n_threads is always n_threads_per_node - should probably make this clear here.

Reply:
I thought Ramble was treating this specially but I think I'm wrong. I meant it to represent OMP_NUM_THREADS and I agree a more-descriptive name would be better (I'm thinking num_omp_threads_per_task)